features/lwipstack/mbed_lib.json

```diff
@@ -132,6 +148,11 @@
             "tcpip-thread-stacksize": 1328,
             "default-thread-stacksize": 640,
             "ppp-thread-stacksize": 896,
+            "memp-num-tcp-seg": 32,
+            "tcp-mss": 1460,
```

Comment:
Should this be at max 1440 to make space for ipv6 header which fixed size is 40 bytes.